src/CommandQueue.ts

```diff
@@ -65,6 +80,9 @@ export class CommandQueue {
             } else {
                 throw new Error('IncorrectParametersException');
             }
+        },
+        addPlugin: (plugin: Plugin) => {
+            this.orchestration.addPlugin(plugin);
```

Comment:
issue: We have an interface for adding plugins in the [`AwsRumConfiguration` object](https://github.com/aws-observability/aws-rum-web/blob/main/src/orchestration/Orchestration.ts#L72) ([example usage](https://github.com/aws-observability/aws-rum-web/blob/main/src/loader/loader-js-error-event.js#L8)). 

Is there benefit for having this as well?

Reply:
Good point. One case that would be beneficial of having an `addPlugin` method is that it gives the users the flexibility to initialize a plugin at any point in time. Initializing plugins only via the config does not allow that as far as I know.

Reply:
Plugins lifecycle management happens through enable, disable and update, which I think covers any situation where `addPlugin` would be used. Since there is no `deletePlugin`, the expectation is that plugins live for the duration of the web client instance.